packages/editor/src/components/provider/index.native.js

```diff
@@ -70,6 +72,15 @@ class NativeEditorProvider extends Component {
 			this.post.type,
 			this.post
 		);
+		this.getEditorSettings = memize(
+			( settings, capabilities ) => ( {
+				...settings,
+				capabilities,
+			} ),
+			{
+				maxSize: 1,
+			}
+		);
```

Comment:
I added the capabilities into the editor settings that are passed to the `EditorProvider`, this is required because we need them when executing the `useNativeBlockEditorSettings` hook.

Additionally, to prevent extra re-renders, I decided to memoize the editor settings object.

Reply:
@fluiddot I was looking around the codebase for some context about what `maxSize` does but I wasn't able to figure it out. Could you please expound a bit on its usage?

Reply:
Is it utilized to define the maximum size of the cache?

Reply:
Exactly, this parameter controls the size of the cache, [here](https://github.com/aduth/memize) you can check the documentation of the memize package.

Reply:
There’s a similar logic in this file:
https://github.com/WordPress/gutenberg/blob/a88a3b587cf203c62c196332d64b33afe164d283/packages/edit-post/src/editor.native.js#L39

Reply:
Good good. Thanks for confirming 🙇🏾